Scatterplots/Scatterplots_30days.rmd

```diff
@@ -0,0 +1,221 @@
+---
+title: "Cumulative Indicator Scatterplots: 30-Day After First Case"
```

Comment:
Are all the files for ndays basically the same? Can the logic be put into a helper function from an R file that just takes in the datatable or something? Maybe not worth it to do those changes now since this is more exploratory, but this seems like a lot of code that might be doing basically the same thing

Reply:
I agree, since this is more exploratory I don't see the need to redo it unless anyone feels strongly about this.